sdk/python/mypy.ini

```diff
@@ -10,6 +10,9 @@ ignore_missing_imports = True
 [mypy-dill]
 ignore_missing_imports = True
 
+[mypy-pulumi.runtime.invoke]
```

Comment:
This will mask all errors of this category for the entire package. I think what we actually want to do is ignore this error for the `google.protobuf.pyext._message` package (or some form of that import). There is some explanation of how this works here: https://mypy.readthedocs.io/en/stable/config_file.html#examples